extensions/amp-skimlinks/0.1/skim-options.js

```diff
@@ -147,6 +150,30 @@ function getInternalDomains_(docInfo) {
   return internalDomains;
 }
 
+/**
+ * Use CNAME domain defined in "custom-redirect-domain" option if specified
+ * or "https://go.skimresources.com" by default.
+ * @param {!Element} element
+ * @return {string}
+ */
+function getWaypointBaseUrl(element) {
+  let customSubDomain = element.getAttribute('custom-redirect-domain');
+  if (customSubDomain) {
+    // Remove potential HTTP protocol.
+    customSubDomain = customSubDomain.replace(/^\/\/|^https?:\/\//, '');
+    // Remove potential trailing slash.
+    customSubDomain = endsWith(customSubDomain, '/') ?
+      customSubDomain.slice(0, -1) :
+      customSubDomain;
+
+    // Use http since publisher CNAME to go.redirectingat.com does not support
```

Comment:
@alabiaga, it's actually the opposite. `go.redirectingat.com` and `go.skimresources.com` which are the default options are both Skimlinks domains (pointing to the same server) and both support `https`. 

However, some publishers prefer to redirect to their own domain name to not confuse their audience. In order to do that, their setup is usually a CNAME pointing to our domain `go.redirectingat.com`.
So when a user goes to, let's say `go.mypublishername.com` (a CNAME set up by the publisher pointing to `go.redirectingat.com`), the user really ends up on `http://go.redirectingat.com` but sees `http://go.mypublishername.com` in the URL. 

So only the publishers using `custom-redirect-domain` (very few) will have to use `http` instead of `https` redirect. 
Supporting `https` for our publishers using a CNAME is something we already had a look into but is impossible for now since we would need to transfer SSL certificates between two domains owned by two different entities (Skimlinks & the publisher).

Reply:
sounds good. thanks for the details.